### DISCUSSION TITLE
Quick Settings menu does not turn off

### DISCUSSION THREAD

**Discussion by Chris-de-ze** [valinet/ExplorerPatcher#1771] [Q&A] [answered]
I searched the discussions and it doesn't appear  to exist a similar announced problem.

My setup: Gigabyte laptop Model G5 MD-51US113SO,   i5-11400H @ 2.70GHz, Windows 11 Pro, Version 22H2,  OS build  22621.2070
I have installed ExplorerPatcher 22621.1992.56.1 (the latest version).

Windows 11 has a   Quick Settings menu on the system tray (please see the attached screenshot). 
If I turn it on, it works perfectly BUT I cannot turn it off and have it disappear from the screen. 
It stays permanently there on top of all other windows covering a part of the monitor screen without being turned off. 
The Only way to take the menu off the screen is to restart Explorer.

Please help. Is there any solution to the above problem? Thank you.

![Quick Settings menu capture1](https://github.com/valinet/ExplorerPatcher/assets/141184433/e0620c1d-e3bc-4142-a3b8-5bdf5f3fb87f)

**Comment by pyrates999** [comment]
Please post windows build.

**Comment by Chris-de-ze** [comment]
Windows 11 Pro, Version 22H2, OS build 22621.2070

I am still NOT able to take the "Quick Settings" menu off the screen, despite the efforts I did ! 
It stays permanently there on top of all other windows.
BTW, the "Quick Settings" menu works perfectly. 
I have to restart Explorer every time to make it disappear.

I love ExplorerPatcher and I don't want to uninstall it. 
I consider it as one of the most  useful apps for Win11.

**Comment by pyrates999** [comment, marked as answer]
That is a preview update: https://support.microsoft.com/en-us/topic/july-26-2023-kb5028254-os-build-22621-2070-preview-66161ea6-c106-43e9-bef5-8410ccbd34e8

Uninstall it if you want to use explorer patcher and don't install preview updates in the future: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

**Comment by Chris-de-ze** [comment]
YOU ARE ABSOLUTELY RIGHT AND THANK YOU VERY MUCH FOR THE PRICELESS SUGGESTION.

There was a windows preview update installed:   _2023-07 Cumulative Update Preview for Windows 11 Version 22H2 for x64-based Systems (KB5028254)_

I uninstalled it and voilà the Quick Settings menu (Win 11 Control Center) turns on and off normally without any problem.

I unclicked also the "Get the latest updates..." 

**THANK YOU VERY MUCH FOR YOUR HELP!!**

**Reply by pyrates999** [reply]
You're welcome!

**Comment by pyrates999** [comment]
Please mark my suggestion as the answer then.

**Comment by Chris-de-ze** [comment]
Done! Thank you very much again
